.babelrc

```diff
@@ -13,6 +13,15 @@
       "plugins": [
         "rewire"
```

Comment:
Do we still need rewire in development if it's in the test?

Reply:
Unfortunately we rewire in files that are tested by Karma and Jest and Karma uses "development" and Jest uses "test".